staging/inputs.nix

```diff
@@ -0,0 +1,14 @@
+# This gives Flakes vibes for everyone who are missing them.
+{ ... }:
+let
+  sources = import ./npins;
+in
+{
+  imports = [
+    "${sources.agenix}/modules/age.nix"
```

Comment:
I guess we also need age in the dev environment? Or at least proper instructions how to set up your system such that it works. Almost sure I won't be able to work with this without painstakingly figuring it out. Not a blocker for this PR, but at least some lines of documentation with pointers will help me making the cleanup a lot faster.

Reply:
I can add agenix in the parent development shell or introduce a deployment specific development shell, your pick.

Reply:
Let's have it all in one devshell

Reply:
OK.